frontend/components/text-classifier/results/RecordExplain.vue

```diff
@@ -62,7 +62,9 @@ export default {
           /* eslint-disable no-mixed-operators */
           const colorSensitivity = 1; // color sensitivity (values from 1 to 4)
           const logNumber = 100 / Math.log10(100) ** colorSensitivity;
-          percent = Math.round(Math.log10(percent) ** colorSensitivity * logNumber);
+          percent = Math.round(
+            Math.log10(percent) ** colorSensitivity * logNumber
+          );
```

Comment:
performance : you can breaks these operations in several computed in order to vue to only computed the one where the dependencies are changing